chpl/chpl-service/src/test/java/gov/healthit/chpl/validation/listing/reviewer/ListingStatusAndUserRoleReviewerTest.java

```diff
@@ -12,8 +12,8 @@
 
 import gov.healthit.chpl.domain.CertificationCriterion;
 import gov.healthit.chpl.domain.CertificationResult;
-import gov.healthit.chpl.domain.CertificationStatus;
 import gov.healthit.chpl.domain.CertificationStatusEvent;
+import gov.healthit.chpl.domain.CertificationStatusProvider;
```

Comment:
We're effectively making a "mock" domain object, right? Maybe worth passing in the ID & effective date as well, and then the Builder can make one with all of the properties? As it stands, the builders below are still passing in "magic" variables in the form of the ID and effective date. I know they're not flagged as "magic", but we know they actually are. Thinking of the class as something that we could call out as a mock as well; wouldn't want it to get confused with "real" code; should it be a "CertificationStatusMock" as opposed to a "CertificationStatusProvider"?

Reply:
But these are not "mock" objects.  They are real object, just not populated by the DB.  They have all of the functionality of a real `CertificationStatus` object.  If we were to pass in an `id` and `effectiveDate` this would be producing `CertificationStatusEvent` objects.  The `CertificationStatusProvider` was designed to provide access to "static" list of statuses - basic lists of data.  I don't think `CertificationStatusEvent(s)` are static, since they are specific and different for every certified product.  That is why I wrote the helper method `getCertificationStatusEvent`.